### ISSUE TITLE
Add "PH=off"-option to complete "opening_hours" quest

### ISSUE THREAD

**Issue by Nordpfeil** [westnordost/StreetComplete#1825]
### General
Quest for Opening Hours
When answering an opening hours quest, I can select all days Monday to Sunday and also "Public Holidays" and add the individual open- and close-times.
But there is no posibility to set "PH=off" to overrule the normal opening hours (for example: Mo-Fr 08:00-18:00). With Street Complete I can only overrule the Mo-Fr opening hours by specific PH opening hours. 

### Checklist
Checklist for quest suggestions (see guidelines):
- [X] 🚧 To be added tag is established and has a useful purpose
- [X] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [X] 🐿️ Easily answerable by everyone from the outside but a survey is necessary 
- [X] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
- [X] 🕓 Applies to a reasonable number of elements (Worth the effort)

### Ideas for implementation
Add a check box [  ] the user can mark with a cross [X] to indicate that it is closed on Public Holidays

**Comment by HolgerJeromin** [comment]
Duplicate of declined #276

**Comment by westnordost** [comment]
The discussion in #276 was a bit long-winded and didn't get a good start because I came of as pretty rude at the beginning.

To summarize the main reason why this possibility does not exist in StreetComplete is that the users ought to enter the opening hours in the way as they see them on the sign. Opening hours signs don't usually say when the shop is closed, but when it is open, so you'll probably never see a "closed on public holidays" on a sign, only "open on public holidays, except this and that day". And that makes sense of course, because that is kind of the default, in the real world.
Some validators for opening hours turned this around in the OpenStreetMap world and seem to assume against common sense that if it is not explicitly given, it means that the shop is open during public holidays, or at least there is a strong doubt about whether it is or is not. But remember, that's just a validator, we don't tag for the validator, it is just a friendly reminder for if we perhaps forgot something.

StreetComplete users in general don't know about these validators, but they know what is on the sign, and can enter it in the app. If there was a checkbox like you suggest, it is the question how many people would actually check the `PH=off` box for normal shops because users of the app are instructed to answer what they see. That a normal shop is closed during public holidays is usually not written on the sign.
So, apart from the doubt that the checkbox would be used in the way you'd like by users, there is also an issue with verifiability with the idea to always specify PH also when they are not `on` - did the mapper add it because of a validator, because he was assuming that as this is a normal shop, it will be closed on public holiday, as required by law? Or did the sign actually mention this?

**tl;dr:** Users should map what they see, that a shop is explicitly closed during PH is usually not something one can see on an opening hours sign. That validator rule that reports a missing `PH=off` should be taken with a grain of salt because it encourages not to map the on-the-ground verifiable truth.

**Comment by matkoniecz** [comment]
This functionality appears to be added in latest not-yet-released version. If you added already some opening hours rule you now can add also off days.
